library/ssl_ciphersuites.c

```diff
@@ -358,12 +358,12 @@ static const mbedtls_ssl_ciphersuite_t ciphersuite_definitions[] =
 #if defined(MBEDTLS_CIPHER_MODE_CBC)
     { MBEDTLS_TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA, "TLS-ECDHE-ECDSA-WITH-AES-128-CBC-SHA",
       MBEDTLS_CIPHER_AES_128_CBC, MBEDTLS_MD_SHA1, MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA,
-      MBEDTLS_SSL_MAJOR_VERSION_3, MBEDTLS_SSL_MINOR_VERSION_1,
+      MBEDTLS_SSL_MAJOR_VERSION_3, MBEDTLS_SSL_MINOR_VERSION_3,
```

Comment:
Note: I don't think any of the two changes I'm suggesting are blocking for this PR. If you want to do them in this PR, that would be great, otherwise, please feel free to ignore them for now, but then please create an issue (in the 3.x EPIC) so that we remember to do that later.

Reply:
A new issue (#4769) has been opened - please go there with any comments re. this issue.